test/externalId.js

```diff
@@ -2,7 +2,7 @@
 
 const { it, describe } = require('mocha');
 const assert = require('assert');
-const ID = require('../lib/externalId');
+const ID = require('../dist/externalId');
```

Comment:
Proč v testech requirujeme z distu namísto z libu? (to platí obecně, ne jenom u tohoto konkrétního require)

Pak přestane fungovat code coverage pro zdrojáky a taky to nutí před každým testem celý rebuildnout.

Reply:
no, vzásadě jde o to ze ten cely projekt je nyni kombinace js/ts... testy jsou v JS, nektere moduly jsou již v TS a na to aby se spustily musi byt transpilovane do JS